### DISCUSSION TITLE
Cinemachine dolly tracks?

### DISCUSSION THREAD

**Discussion by ZenRepublic** [ramokz/phantom-camera#211] [Ideas]
I really liked the tracks in cinemachine.. like creating the rails using points and have the camera move along the trail. I saw there's something similar with path, but I tested it out and it doesn't seem to work the same...

Am I doing something wrong or are the tracks not implemented yet?

**Comment by ramokz** [comment]
The current version of `Path Follow` should be working as intended. There aren't any pending changes for it, nor been any complaints about adding more thus far.

What isn't working for you?

**Reply by ZenRepublic** [reply]
Hmm, i couldn't get curved paths to work right, but maybe that's just an oversight on my part. Ill try again

**Reply by ramokz** [reply]
What isn't working right? Have a hunch for what you might be running into.
